Fix docs typo in Bitraversable composition law

ploeh opened the pull request:
Please accept my apologies for this minuscule pull request, but I feel _fairly_ certain that the documentation contains a bug or typo.

Yesterday, I was writing some Quickcheck properties to check the `Bitraversable` laws for an instance I'd written, and I noticed that, as given in the docs, the _composition_ law didn't compile. Changing `traverse` to `bitraverse` not only made my code compile, it also makes sense when thinking about what the law states.

bgamari commented:
Yes, good catch! I'll merge.

Don't worry about the CI failures; this has been fixed since you opened the PR.

Icelandjack commented:
Can you indent it by two spaces that removing `bi` left?

ploeh commented:
> Can you indent it by two spaces that removing `bi` left?

I'm not sure I follow... I didn't remove `bi`, I added it.

Will indentation have any effect either way. In [the currently rendered documentation](http://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Bitraversable.html), at least on my machine, it looks like this:

![image](https://user-images.githubusercontent.com/242165/47769566-9d9c9280-dcdc-11e8-9b8a-4083b521db7b.png)

I've highlighted the part that I've changed.

bgamari commented:
I've merged this in 7e1690d17b39a9491e46b12297adad8d0c89b550.

Icelandjack commented:
I meant adding
